.circleci/config.yml

```diff
@@ -268,11 +268,7 @@ jobs:
   test_benchmark_smoke:
     executor: node-browsers
     steps:
-      # TODO [#3691]: remove `apt-get` and `chrome-version` lines when CircleCI issue is fixed
-      # https://github.com/CircleCI-Public/browser-tools-orb/issues/90
-      - run: sudo apt-get update
-      - browser-tools/install-chrome: # required for selenium used by tachometer benchmark smoke tests
-          chrome-version: 116.0.5845.96
+      - browser-tools/install-chrome # required for selenium used by tachometer benchmark smoke tests
```

Comment:
No longer needed, and needs to be removed so we can use the correct chrome 117 version.